scripts/distribution/ubuntu-packages/template/scripts/concordium-node-wrapper

```diff
@@ -7,171 +7,27 @@ export LD_LIBRARY_PATH=/usr/local/lib
 
 ARGS=""
 
-# Determine what arguments to pass to the binary
```

Comment:
Can we get rid of this alltogether please?

If we need to create the two directories in the install script we should do that. I believe the data directories is already created, and it should not be hard to do that, see https://man7.org/linux/man-pages/man1/dh_installdirs.1.html

Just need to add a `concordium-node.dirs` file inside the `debian` directory naming the config dir.